common/src/main/java/com/microsoft/identity/common/nativeauth/internal/controllers/NativeAuthMsalController.kt

```diff
@@ -135,12 +132,14 @@ class NativeAuthMsalController : BaseNativeAuthController() {
                 oAuth2Strategy = oAuth2Strategy,
                 parameters = parameters)
 
-            if (parameters is SignInStartUsingPasswordCommandParameters)
+            val hasPassword = parameters.password != null && (parameters.password!!.isNotEmpty())
```

Comment:
use `parameters.let{}` to avoid `password!!`

Reply:
Done